apps/coscrad-frontend/src/store/slices/auth/types/authenticated-user-info.ts

```diff
@@ -0,0 +1,9 @@
+export type AuthenticatedUserInfo = {
+    /**
+     * Note that the actual user (profile, etc.) is derived
+     * state and should be accessed by joining in data from
+     * the user slice on the `userId` from the auth provider.
+     */
+    userId: string;
```

Comment:
Or maybe this won't work. We need to find a way to get the userId from auth0 if we'll do it this way.

Reply:
Done.